arkouda/pdarrayclass.py

```diff
@@ -856,7 +1048,7 @@ def is_sorted(pda : pdarray) -> bool:
     else:
         raise TypeError("must be pdarray {}".format(pda))
 
-def sum(pda : pdarray) -> float:
+def sum(pda : 'pdarray') -> float:
```

Comment:
could return a float64 or int64 depending on dtype of input pdarray

Reply:
fixed!

Reply:
returns np.int64